manifest.json

```diff
@@ -588,4 +610,4 @@
          "version" : "0.1"
       }
    ]
-}
+}
```

Comment:
Could you revert this newline removal?
Ideally files should end with a newline, but I'd really like to keep it at least in the manifest.